place_tracker/lib/place_details.dart

```diff
@@ -235,4 +246,88 @@ class _Map extends StatelessWidget {
       ),
     );
   }
+}
+
+class _Reviews extends StatelessWidget {
+
+  Widget _review(String reviewText) {
```

Comment:
Consider a more specific name here. Some of our other examples use things like `_buildSingleReview` or `_buildReviewWidget` to more clearly indicate what's going on.